plugins/reporters/spdx/src/main/kotlin/SpdxDocumentModelMapper.kt

```diff
@@ -43,20 +43,11 @@ import org.ossreviewtoolkit.utils.spdxdocument.model.SpdxRelationship
  * A class for mapping [OrtResult]s to [SpdxDocument]s.
  */
 internal object SpdxDocumentModelMapper {
-    data class SpdxDocumentParams(
-        val documentName: String,
-        val documentComment: String,
-        val creationInfoComment: String,
-        val creationInfoPerson: String,
-        val creationInfoOrganization: String,
-        val fileInformationEnabled: Boolean
-    )
-
     fun map(
         ortResult: OrtResult,
         licenseInfoResolver: LicenseInfoResolver,
         licenseTextProvider: LicenseTextProvider,
-        params: SpdxDocumentParams
+        config: SpdxDocumentReporterConfig
```

Comment:
Could we instead turn this `SpdxDocumentParams` into an `interface, and make the plugin configuration implement it? (This would ensure this mapper stands on its own, and takes as input only what is actually needed)